docs/running-clusterfuzzlite/gitlab.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: default
+parent: Step 2&#58; Running ClusterFuzzLite
+grand_parent: ClusterFuzzLite
+title: GitLab
+nav_order: 1
```

Comment:
Make this nav order 2 please and make google cloud build 3 and prow 4.

Reply:
ok